arch/arm/boot/dts/overlays/dt-blob-overlay.dts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Device tree blob overlay for Revolution Pi by KUNBUS
+ *
+ * RevPi Devices
+ */
+
+#include "rpi-dt-blob-overlay.dts"
+
+
+/ {
+	videocore {
+		pins_cm {
+			pin_config {
+				/* For Core SPI0 CLK */
+				pin@p39 { function = "spi"; termination = "no_pulling"; drive_strength_mA = < 8 >; };
+			}; // pin_config
+			pin_defines {
+			}; // pin_defines
+		};
```

Comment:
Is this really necessary? CM1 devices will always have their full blown overlay files. The minimal dt-blob is only used on newer devices which have a HAT eeprom (and can't configure the drive strength from there atm)

Reply:
yes, you are right, I think it's not needed.